### ISSUE TITLE
implement CV/pitch quantizer module

### ISSUE THREAD

**Issue by robertoostenveld** [eegsynth/eegsynth#33]
It should implement "pitch quantization" according to ttps://en.wikipedia.org/wiki/Quantization_(music).

It could work with an ini file like this

```
[input]
channel1=brain.occipital.alpha
channel2=brain.frontal.theta

[quantization]
level=1,2,3,4,5,...       ; this would be required, input is mapped onto this
quantized=1,2,3,4,5,...   ; all following ones are the optional outputs
octave=8,9,10,11,...  
fifth=?,?,?,...           ; see https://en.wikipedia.org/wiki/Perfect_fifth
```

the output would be polyphonic and consist of
brain.occipital.alpha.quantized
brain.occipital.alpha.octave
brain.occipital.alpha.fifth
etc.

**Comment by robertoostenveld** [comment]
There is actually some overlap in functionality with the (very generic) post processing module, which can also quantize on the basis of equations. 

See https://github.com/eegsynth/eegsynth/issues/32#issuecomment-200248053 for an example.

**Comment by robertoostenveld** [comment]
the proposed functionality here is different in the respect that it is not doing a mathematical operation, but only table lookups. The input and output table could also be very different. E.g. the suggested quantizer module could also be used to transpose, to limit, or compress.

**Comment by robertoostenveld** [comment]
mac011> git commit
[master c2bbabe] implemented quantizer module, see issue #33
 4 files changed, 176 insertions(+)
 create mode 100644 module/quantizer/README.md
 create mode 100644 module/quantizer/quantizer.ini
 create mode 100755 module/quantizer/quantizer.py
 create mode 100755 module/quantizer/quantizer.sh

fixed!
